### DISCUSSION TITLE
Don't Force Noto Sans

### DISCUSSION THREAD

**Discussion by mariocynicys** [community/community#39226] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Yesterday github's font used to look like this (JetBrains Mono):
![image](https://user-images.githubusercontent.com/56920956/202123421-6e587097-fe06-43c7-bc73-36d50f0081f5.png)

But now it looks like this (Noto Sans):
![image](https://user-images.githubusercontent.com/56920956/202125203-9ca08c95-002a-4e8c-9500-e9e378ce38cb.png)

This is because of the font chosen here:
![image](https://user-images.githubusercontent.com/56920956/202123969-c441cfa2-2d77-4d58-b62f-b27347f995a5.png)

I think sans-serif used to be the first on the list, which I have it mapped to JetBrains Mono on my system.
Better not force a specific font for people and let them choose their own based on their systems (IMO).

Please revert this change or if anybody can tell how to remove the font attribute from a specific site???

**Comment by Preetiraj3697** [comment, marked as answer]
You can also use an extension call Stylebot([link](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha?hl=en)) on Chrome. It's not just limited to font changing but allow you to inject custom css to it.

**Comment by ganigeorgiev** [comment]
Today I also noticed the font change. In addition that the font sizing now doesn't look consistent everywhere, it also breaks for some reason the backtick character. Here is how it looks on Chrome 106, Linux (Fedora):

![screenshot](https://i.imgur.com/rw70mAl.png)

**Reply by ganigeorgiev** [reply]
I've just found that you can force a monospace font for the markdown editor that "fixes" the backtick overlap. You can go to https://github.com/settings/appearance and enable the "Markdown editor font preference" at the bottom of the page:

![settings_screenshot](https://i.imgur.com/NcBhrSj.png)

_It still looks over-sized though._

**Reply by simurai** [reply]
The backtick issue with `Noto Sans` should now be fixed with [this workaround](https://github.com/notofonts/noto-fonts/issues/736#issuecomment-628914669).

**Reply by austinbutler** [reply]
GitHub has fixed this now.

**Reply by ganigeorgiev** [reply]
@austinbutler The backtick issue is still there, but as @simurai linked above, the issue is caused by an old/buggy local Noto Sans version. Updating the local font fixes it.

**Reply by junaruga** [reply]
Thanks! The workaround checking "Use a fixed-width (monospace) font when editing Markdown" works for me on Firefox on Fedora 36!

**Comment by jgneff** [comment]
I love the new font! I'm on Ubuntu 20.04.5 LTS (Focal Fossa). If I remember correctly, it was matching Helvetica before and providing Nimbus Sans, which looked rather awful. I'm not certain, though. I just remember it was fuzzy, small, cramped, and difficult to read,

Now I'm getting a direct match for Noto Sans, and I can finally read the text in my issues and pull requests!

So just to add another conflicting opinion: please **do not** revert this change! :smile: Or rather, please make sure to provide a good modern font for Linux. I even liked the short-lived Ubuntu system font that showed up for a few days about a year ago. The Ubuntu font looked great, but this change is still a big improvement.

**Reply by mariocynicys** [reply]
If the font were to be set to some general family like sans or sans-serif users can then have their default system fonts displayed, instead of forcing a specific font.

Don't dislike noto sans as well, but this is not how I used to see github.
[this](https://github.com/orgs/community/discussions/39226?notification_referrer_id=NT_kwDOA2SLfLM0ODQ4MDgyNDUxOjU2OTIwOTU2#discussioncomment-4155234) fixed the issue for me anyways.

**Comment by LegionMammal978** [comment]
Note that `sans-serif` was always at that position in the list: the only change was that `"Noto Sans"` was inserted before it. On my Ubuntu 22.04.1 LTS system (upgraded from 18.04 LTS) with Firefox, `sans-serif` was the first family matched. `sans-serif` has always defaulted to DejaVu Sans on my system, which I definitely prefer to Noto Sans.

**Reply by jgneff** [reply]
Yes, the DejaVu Sans match on `sans-serif` looks great, too. I would agree that it's even better than Noto Sans. Now that I played around with it some more, it really was just the Nimbus Sans (NimbusSans-Regular) match for Helvetica that I found too small and cramped. Real Helvetica looks good on macOS, but the substitutes on Linux aren't quite as nicely rendered. Helvetica probably gets in the way for me because of all the TeX fonts I installed.

I assume GitHub added the Noto Sans font for all the Unicode coverage it offers.

**Comment by Frefreak** [comment]
I noticed this too. My complain to Noto Sans is that the bold font seems too bold to me and doesn't look as beautiful as before to me.

It stands out when a lot of bold fonts are involved:
![image](https://user-images.githubusercontent.com/7122156/202417272-5145a189-58ad-449c-bded-962f9869a5e8.png)
![image](https://user-images.githubusercontent.com/7122156/202417770-f2882209-daa1-4244-9474-ca1bb20d7c99.png)

I personally prefer the old way (sans-serif is matched, which maps to source sans pro in my system)

![image](https://user-images.githubusercontent.com/7122156/202417456-2590433a-24bf-43be-a0b1-2d533227dfb3.png)
![image](https://user-images.githubusercontent.com/7122156/202417980-8921d9dd-ad85-4d2e-bb2e-140d9a30f082.png)

**Comment by fionn** [comment]
I "solved" this problem for myself by uninstalling `noto-fonts`, but I wish I didn't have to.
